images/win/scripts/Installers/Validate-AzCopy.ps1

```diff
@@ -0,0 +1,7 @@
+if (Get-Command -Name 'azcopy') {
+    Write-Host "azcopy $(azcopy --version) on path"

```

Comment:
We should remove duplicated azcopy word:
```
PS > "azcopy $(azcopy --version) on path"
azcopy azcopy version 10.4.3 on path
```

Reply:
Fixed